lib/player.rb

```diff
@@ -7,6 +7,7 @@ def initialize(name)
       @name = name
       @plays = []
       @won = false
+      @player_tiles = []
     end
 
     def name
```

Comment:
You have an `attr_accessor` for the `@name` and `@plays` instance variables so this method and the method below are not necessary.